tutorials-v5/visualization/animation-demo.md

```diff
@@ -0,0 +1,214 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Animation demos
+
+For more information about QuTiP see [http://qutip.org](http://qutip.org)
+
+```python
+from qutip import (ket, basis, sigmaz, tensor, qeye, mesolve,
+                   complex_array_to_rgb, about)
+import qutip as qt
+import numpy as np
+import matplotlib.pyplot as plt
+```
+
+
+```python
+# if this does not work on your environment, use qt.make_html_video
+# to see animations.
+%matplotlib notebook
+```
+
+## Time evolution of an operator
+
+
+```python
+# Hamiltonian
+H = qt.sigmay().unit()
+
+# initial state
+psi0 = qt.sigmax().unit()
```

Comment:
I would change this example to something like:

```
psi0 = basis(2,0)
tlist = np.linspace(0, 2 * np.pi, 50)
c_ops = [np.sqrt(0.1) * destroy(2).dag()]
results = mesolve(H, psi0, tlist, c_ops, [])`
```

And perhaps add a little more explanation